### PR TITLE
MOTECH-2203 Fixed the error in IS_TRUE filter

### PR DESCRIPTION
**Pull request by shanmukhm**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/OperatorType.java [hunk 1]
```diff
@@ -59,7 +59,9 @@ public static boolean needExpression(String string) {
         OperatorType type = fromString(string);
         boolean need = true;
 
-        if (type != null && (type == EXIST || type == AFTER_NOW || type == BEFORE_NOW)) {
+        if (type != null && (type == EXIST || type == IS_TRUE)) {
```
Comment (shanmukhm) [inline modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/OperatorType.java]: I had to split it because of a checkstyle rule

**Comment by pgesek** [comment]
test this please

**Comment by pgesek** [comment]
test this please

**Pull request merged**